main.go

```diff
@@ -79,8 +80,15 @@ func main() {
 		defer cancelFunc()
 
 		globalLogger.Info().Msg("Starting coraza-spoa")
+
```

Comment:
This sounds really cool as a feature but I would really like to reduce our external dependencies, especially when the imported functionality is [just 30 lines of code](https://github.com/coreos/go-systemd/blob/main/daemon/sdnotify.go) :smile: Could you please do this without the library?